tests/integration_tests/test_applications_from_config.py

```diff
@@ -172,9 +179,13 @@ def write_and_get_tmp_config_file(config, output_path):
     Returns
     -------
     str: path to the temporary config file.
+    str: configuration string.
 
     """
 
+    if len(config) == 1 and next(iter(config.values())) is True:
+        return None, "--" + list(config.keys())[0].lower()
```

Comment:
I think we use f-strings in logger messages, but we don't have a rule in general. The code is full of this simple string additions.

Reply:
And I have complained about it many times already. String additions should be avoided as much as possible (i.e., almost always). We use f-string everywhere else.

Reply:
We have hundreds of simple string1 + string2 in the code.

Do we really need that discussion? I can understand the improvement in readability with complicated string compositions, but for string1 + string2??

Reply:
Old code I fix whenever I run into it. New code should use f-string so that it's not an infinite loop of fixing. The reason is not readability but of speed, convention and consistency. 

We've had this discussion in the past already and came to an agreement. I even ran a script once that went through the entire code to fix it.

Reply:
Resolved this discussion today in zoom.